ios/FluentUI/DrawerVnext/SlideOverPanel.swift

```diff
@@ -95,7 +129,7 @@ struct SlideOverPanel<Content: View>: View {
         return slideOutPanelWidth * contentWidthSizeRatio
     }
 
-    public var body: some View {
+    var body: some View {
```

Comment:
nit: this is internal by default if you don't have a specifier, then it should be moved up so the private props are the last thing in the file? (Public -> Internal -> Private)

Reply:
thanks for catching this, I have updated the ordering across the module